src/assets/sass/protocol/components/_zap.scss

```diff
@@ -0,0 +1,24 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+@import '../protocol/includes/lib';
+
+[class*=" mzp-t-zap"], [class^=mzp-t-zap] strong {
```

Comment:
Yeah this is probably trying too hard to avoid collisions and I'll simplify to just the contains selector. We can re-write the rule if it conflicts.

I'm more worried about if I'm violating the exceptions for how the design system should work since in all other cases we use one class to set the thing and a second to modify it.